index.html

```diff
@@ -4049,6 +4049,18 @@ <h3>Prefer Single-Use Credentials</h3>
       </p>
     </section>
 
+    <section>
+      <h3>Private Browsing</h3>
+
+      <p>
+In an ideal private browsing scenario, no personally identifiable information (PII)
+will not be revealed. Because many <a>credentials</a> include PII, organizations providing
```

Comment:
@brentzundel shouldn't this be "no personally identifiable information (PII)
will be revealed" or "personally identifiable information (PII) will not be revealed" ?

As it currently reads, it's a double negative so makes it seems like all PII would be revealed in private browsing

Reply:
Yeah, why do you hate privacy, @brentzundel? :P